### DISCUSSION TITLE
Mise en page des textes des descriptions

### DISCUSSION THREAD

**Discussion by christianroy** [culturecreates/artsdata-planet-lavitrine#19] [Ideas]
En observant le dump du GTQ, j'ai vu cette description:

```
      "description": {
        "@none": "Suite au succès notable de son album Chansons hivernales sorti en 2020, Pierre Lapointe présentera ses pièces sur scène lors d’une tournée durant le temps des fêtes. Entouré de ses musiciens et d’invités spéciaux dans un décor à couper le souffle, il mettra de l’avant son répertoire aigre-doux et arrivera assurément à surprendre l’auditoire une fois de plus.Sensibles et rythmées, ses chansons rappelleront les réunions de famille, les amis, les excès, les bilans de fin d’année, mais aussi la tristesse et la solitude qui accompagnent souvent l’arrivée des grands froids. Laissez-vous guider dans l’univers chic et singulier d’un des créateurs les plus importants de sa génération.À propos de Chansons hivernalesChansons hivernales est le onzième album enregistré en studio de l’auteur-compositeur-interprète Pierre Lapointe. Réalisé par Emmanuel Éthier, l&#x27;album de compositions originales a fait ses débuts au numéro 10 du classement canadien des albums en décembre 2020. Ce projet, qu’il caressait depuis une quinzaine d’années, lui a permis d’explorer la légèreté douce-amère de la saison des fêtes.« J’avais le fantasme d’écrire des chansons qui se rattachaient à la période des fêtes en essayant de trouver de nouveaux chemins »- Pierre Lapointe (en entrevue à Radio-Canada)Fan de Pierre? Consultez vite notre chronique Curieux de Culture où il vous parle des artistes qui l&#x27;inspirent. Une belle occasion de plonger dans son univers et de faire des découvertes culturelles!"
      }
```

Il s'agit de la description de l'événement de cette page: https://grandtheatre.qc.ca/programmation/pierre-lapointe-chansons-hivernales/

Le problème est que les changements de ligne sont perdus dans la description. Par exemple, le passage `...sa génération.À propos de Chansons hivernalesChansons hivernales...` aurait besoin de changements de ligne.

**Comment by saumier** [comment]
@christianroy I agree this is problematic. The description in the dump is exactly the description created by GTQ in their JSON-LD.  I have come across this problem many times.  

Ideas:
1. The text could maybe go thought a second process to "massage" it and add back some basic `<p>` layout. 
2. In the past we have often resorted to scraping the HTML for the description instead of the JSON-LD  I took a look, and I see that the entire div class "rich-text" could be scraped to include the HTML. Since we already scrape the HTML to get the "event types" along with the JSON-LD, I could add this additional scrape.
3. Switch to use Footlight to scrape. So with Footlight we can get the layout in the description text and preserve HTML.
4. Use the og:description which is very vey short but at least looks OK for the couple I looked at.

**Reply by saumier** [reply]
As discussed this morning (Christian, Genevieve) lets go with option 2 as a low priority after Festivals and other main issues are resolved.  I opened issue https://github.com/culturecreates/artsdata-planet-lavitrine/issues/21
